src/components/Dropdown.jsx

```diff
@@ -1 +1,35 @@
-// TODO (Togzhan & Yi & Aryaa): Create a dropdown component
\ No newline at end of file
+import React, { useState } from 'react';
+
+const Dropdown = ({ label, children }) => {
+  //use isOpen to keep track of whether the dropdown menu is visible
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <div 
+      // style below for menu
+      class="absolute inline-block"
```

Comment:
Please change the `class` attribute to `className`. In React we don't want to use `class` because that's a reserved keyword in JavaScript and we don't want to mix that up with the HTML attribute. Instead, to assign classes in React we use the `className` attribute. If you look in the console, there's an error.
<img width="501" alt="image" src="https://github.com/user-attachments/assets/a6c79c16-d174-4680-8d1d-4fac58e40dcf">

Some browsers are lenient with interpreting `class` as `className` which is why the styling still works in Chrome, but it can lead to issues in the future.